src/presets/universal_deployer.cairo

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts for Cairo v0.10.0 (presets/universal_deployer.cairo)
+
+/// # UniversalDeployerContract Preset
+///
+/// The Universal Deployer Contract is a singleton smart contract that wraps `deploy_syscall`
+/// to expose it to any contract that doesn't implement it, such as account contracts.
```

Comment:
```suggestion
/// The Universal Deployer Contract is a standardized generic factory of Starknet contracts.
```
i think it's better to focus on what it is instead of the motivation/how it's implemented. the how is available right below.